src/utils.ts

```diff
@@ -1,22 +1,22 @@
-import { QueryObjType, GraphQLResolveInfo } from './types.ts';
+import { FieldNode, CacheKeyObj, GraphQLResolveInfo } from './types.ts';
 
-const queryExtract = (info: GraphQLResolveInfo): string => {
+
+const buildCacheKey = (info: GraphQLResolveInfo): string => {
   if (info.operation.operation === 'mutation') {
     console.error(
       '%cDenostore cache function does not allow caching of mutations.',
       'font-weight: bold; color: white; background-color: red;'
     );
     throw new Error('Query error. See server console.');
   }
-  const node = info.fieldNodes[0];
-  const queryObj: QueryObjType = {
+  const node: FieldNode = info.fieldNodes[0];
+  const cacheKeyObj: CacheKeyObj = {
```

Comment:
should the type "CacheKeyObj" be called "CacheKeyObjType"?

Reply:
Thanks for the catch I actually need to change "QueryObjType" to just "QueryObj"

Reply:
No I don't think the "Type" is necessary because our other defined types don't have "Type" at the end